cross-linux-android/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM debian:stable
+
+ENV DEBIAN_FRONTEND noninteractive
+
+ARG NDK="21"
```

Comment:
Can we either move this file to `cross-linux-android/ndk21/Dockerfile`, or add build & push hooks like in roc-java image? (I'm OK with both approaches; this specific dockerfile is very simple so I think it'd OK to copy-paste it). I'd like to have a tag for each NDK version, so that in future we can check multiple versions in our CI.

Reply:
Ok I'll take care of managing different images for various NDK :+1: